Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -1435,6 +1453,11 @@ void WebProcess::initializeSandbox(const AuxiliaryProcessInitializationParameter
 {
 }
 
+const char* WebProcess::processTypeDescription() const
```

Comment:
Please use ASCIILiteral

Reply:
Deleted this as I wasn't actually using it.